easybuild/easyconfigs/s/Salome/Salome-8.5.0-intel-2020a.eb

```
@@ -0,0 +1,29 @@
+name = 'Salome'
+version = '8.5.0'
```

@ekieffer since this binary is for 64bit centos7 only, this should be indicated with a versionsuffix:
```suggestion
version = '8.5.0'
versionsuffix = '-centos7_x86_64'
```